netdata/netdata
# Cleanup, rename and packaging fix (Windows Codes)

##### Summary
This PR is bringing some adjusts we had talked in our meetings.
It also removes unnecessary codes from our CMakelist.txt.

Considering this PR was built on top of https://github.com/netdata/netdata/pull/18443, I am going to keep it as draft for while.

##### Test Plan

1. Compile this branch
2. Make an installer
3. Update your system and verify that Netdata is running on Microsoft Windows.

##### Additional Information


<details> <summary>For users: How does this change affect me?</summary>
  
</details>
